src/AppInstallerCommonCore/Public/AppInstallerErrors.h

```diff
@@ -57,6 +57,7 @@
 #define APPINSTALLER_CLI_ERROR_INVALID_MANIFEST                 ((HRESULT)0x8A15002A)
 #define APPINSTALLER_CLI_ERROR_UPDATE_NOT_APPLICABLE            ((HRESULT)0x8A15002B)
 #define APPINSTALLER_CLI_ERROR_UPDATE_ALL_HAS_FAILURE           ((HRESULT)0x8A15002C)
+#define APPINSTALLER_CLI_ERROR_REMOVE_INSTALLER_FAILED          ((HRESULT)0x8A15002D)

```

Comment:
>APPINSTALLER_CLI_ERROR_REMOVE_INSTALLER_FAILED [](start = 8, length = 46)

There is a switch statement with error strings that is in the cpp for this file.

Reply:
yea, I can add to that for completeness. But code paths using AICLI_TERMINATE does not invoke that GetFriendlyNameForErrorCode() method. Code paths using AICLI_TERMINATE should handle output message individually.

---
In reply to: [510503586](https://github.com/microsoft/winget-cli/pull/625#discussion_r510503586) [](ancestors = 510503586)

Reply:
I agree, it was simply for completeness.

---
In reply to: [510541016](https://github.com/microsoft/winget-cli/pull/625#discussion_r510541016) [](ancestors = 510541016,510503586)